pyproject.toml

```diff
@@ -1,7 +1,9 @@
 [project]
 authors = [
-  {name = "Robbin Bouwmeester", email = "robbin.bouwmeester@ugent.be"},
+  { name = "Robbin Bouwmeester", email = "robbin.bouwmeester@ugent.be" },
+  { name = "Henry Webel", email = "henry.webel@sund.ku.dk" },
 ] # TODO: Add others
+description = "ProteoBench is a tool for the analysis of proteomics data."
```

Comment:
It is not really for analysing proteomics data. Maybe it would be cleaner to say "ProteoBench allows the comparison of outputs of proteomics data analysis pipelines"?

Reply:
I made it affirmative: "ProteoBench compares the outputs of mass spectrometry-based proteomics data analysis pipelines"